node/src/launcher.ts

```diff
@@ -75,7 +75,7 @@ export function BrowserStackLauncher(
       const session = (await browser.getSession()).getId()
       log.debug(this.id + ' has webdriver SessionId: ' + session)
       browserMap.set(this.id, { browser, session })
-      pageUrl = makeUrl(pageUrl, args.useHttps)
+      pageUrl = makeUrl(pageUrl, args.useHttps ?? true)
```

Comment:
Since the use of the combined server is optional, let's use `false` by default. Also, `undefined` is falsy, so it's natural to have `false` by default. What do you think?